sdk/core/azure-core/src/main/java/com/azure/core/models/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing core model classes.
+ */
+package com.azure.core.models;
```

Comment:
Add this package to the parent pom's JavaDoc groups as well.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/parents/azure-client-sdk-parent/pom.xml#L544

Reply:
Added and fixed some incorrect groups as well